### ISSUE TITLE
[Bug]:  [ERR_MODULE_NOT_FOUND] issue during runtime and during building of application on NextJS

### ISSUE THREAD

**Issue by arjunvijayanathakurup** [blocknative/web3-onboard#1228]
### Current Behavior

When I try `npm run build`, on my NextJS project, I am receiving the module not found error. It was working perfectly previously but now, receiving the module not found error. 

### Expected Behavior

The build should be successful without having any error based on module not found.

### Steps To Reproduce

1. Packages being used in the project: "@web3-onboard/core": "^2.0.11, "@web3-onboard/injected-wallets": "^2.0.2"
2. Tried the latest package with version: "@web3-onboard/core": "^2.8.0", "@web3-onboard/injected-wallets": "^2.1.0",
3. Install using `npm i @web3-onboard/core @web3-onboard/injected-wallets`
4. After installing the package, run build command using `npm run build`
5. Similar error occured for `npm run dev` which starts the local development server

### What package is effected by this issue?

@web3-onboard/core

### Is this a build or a runtime issue?

Build, Runtime

### Package Version

2.0.11

### Node Version

v16.15.1

### What browsers are you seeing the problem on?

Firefox, Chrome

### Relevant log output

```shell
> Build error occurred
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/arjunv/Documents/Work/sampleProject/node_modules/@web3-onboard/common/dist/errors' imported from /Users/arjunv/Documents/Work/sampleProject/node_modules/@web3-onboard/common/dist/index.js
    at new NodeError (node:internal/errors:372:5)
    at finalizeResolution (node:internal/modules/esm/resolve:437:11)
    at moduleResolve (node:internal/modules/esm/resolve:1009:10)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  type: 'Error',
  code: 'ERR_MODULE_NOT_FOUND'
}
```

### Sanity Check

- [X] If this is a build issue, I have included my build config. If this is a runtime issue, I have included reproduction steps and/or a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example).

**Comment by Adamj1232** [comment]
@arjunvijayanathakurup We are working on a fix for this now and we should have alpha version to test  a fix for this by EOD. Thank you for reporting this issue

**Comment by blueporest** [comment]
I have the same isuue.
It is temporarily in use contain the js extension.

**Comment by Adamj1232** [comment]
We have alpha packages released to fix this. If you bump the patch version number and append `-alpha.1` you can pull in the latest. You can see the fix PR(and version changes per package here - https://github.com/blocknative/web3-onboard/pull/1227/files)

**Comment by arjunvijayanathakurup** [comment]
@Adamj1232, I have tested it out and the build as well as runtime errors have been fixed.

**Comment by aaronbarnardsound** [comment]
Fixed and released in version [2.9.0](https://github.com/blocknative/web3-onboard/releases/tag/2.9.0)
